src/sentry/integrations/api/serializers/models/data_forwarder.py

```diff
@@ -0,0 +1,76 @@
+from collections import defaultdict
+from collections.abc import Mapping, Sequence
+from typing import Any
+
+from django.contrib.auth.models import AnonymousUser
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.integrations.models.data_forwarder import DataForwarder
+from sentry.integrations.models.data_forwarder_project import DataForwarderProject
+from sentry.users.models.user import User
+from sentry.users.services.user import RpcUser
+
+
+@register(DataForwarder)
+class DataForwarderSerializer(Serializer):
+    def get_attrs(
+        self,
+        item_list: Sequence[DataForwarder],
+        user: User | RpcUser | AnonymousUser,
+        **kwargs: Any,
+    ) -> dict[DataForwarder, dict[str, Any]]:
+        project_configs = DataForwarderProject.objects.filter(data_forwarder__in=item_list)
+        project_config_attrs = defaultdict(set)
+        for project_config in project_configs:
+            project_config_attrs[project_config.data_forwarder].add(project_config)
+
+        return {
+            data_forwarder: {"project_configs": project_config_attrs.get(data_forwarder, set())}
+            for data_forwarder in item_list
+        }
+
+    def serialize(
+        self,
+        obj: DataForwarder,
+        attrs: Mapping[str, Any],
+        user: User | RpcUser | AnonymousUser,
+        **kwargs: Any,
+    ) -> dict[str, Any]:
+        project_configs = attrs.get("project_configs", set())
+
+        return {
+            "id": str(obj.id),
+            "organizationId": str(obj.organization_id),
+            "isEnabled": obj.is_enabled,
+            "enrollNewProjects": obj.enroll_new_projects,
+            "enrolledProjects": [
+                str(project_config.project_id) for project_config in project_configs
+            ],
```

Comment:
It may be worth serializing the `Project` (or probably just a minimal version of it) object here and in the DataForwarderProject serializer to save an API call & complexity on the frontend. 

Since both endpoints will be paginated, it's not easy to ensure the UI has all the data to render the project (i.e. slug, avatar, etc) that the config applies for. you can imagine an org with 1000 projects, but the project slugs/avatars in the first 100, don't match the ones in `enrolledProjects` -- you'd have to have the frontend make another API call and maybe another after that

Reply:
We're kinda predicting what we'll need for the frontend here, but maybe serializing just [these fields](https://github.com/getsentry/sentry/blob/master/static/app/types/project.tsx#L8-L13) would be a good call! Gives our UI more flexibility, since displaying IDs isn't user friendly.